bayesrules/Section_2_1_Fake_news.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Thanks! This note book is a good start though I'm realizing I should create a template to ensure each chapter is consistent. I will get to that in the next couple of days!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thinking about it lets get this merged and then set it up to be a template afterwards. Should make things quite easy!

Reply:
Sounds great